test/cql/cdc_delta_modes_test.cql

```diff
@@ -0,0 +1,16 @@
+create table tb1 (pk int, ck int, i1 int, PRIMARY KEY (pk, ck)) with cdc = {'enabled': true, 'preimage': false, 'postimage': false, 'delta': 'off'};
+
+create table tb2 (pk int, ck int, i1 int, PRIMARY KEY (pk, ck)) with cdc = {'enabled': true, 'preimage': true, 'postimage': true, 'delta': 'off'};
+-- Should add 2 rows (preimage + postimage)
+insert into tb2 (pk, ck, i1) VALUES (1, 11, 111) USING TTL 1111;
```

Comment:
nitpick (I'll commit this anyway, just asking): Why this "USING TTL 1111"? How is it relevant for this test?